src/main/scala/chisel3/util/Decoupled.scala

```diff
@@ -265,14 +281,23 @@ class Queue[T <: Data](val gen: T,
   }
 
   val ptr_diff = enq_ptr.value - deq_ptr.value
+  val flush = Wire(Bool())
+
+  if (hasFlush) { 
+    flush := io.flush.get //to mitigate returning None from io.flush.get in logic below
+  } 
+  else { 
+    flush := false.B 
+  }
```

Comment:
I would consider
```
flush := io.flush.getOrElse(false.B)
```

Reply:
agree, and this can be a `val flush = ...` right?